### PR TITLE
Point launcher at its new signing key.

### PR DESCRIPTION
**Pull request by samiwel**
This change updates the container environment variables for go-launch-a-survey service to point at its newly created signing key. This key has already been deployed to pre-prod via Jenkins on March 27th.

### CODE REVIEW

**Review by tgandrews — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by samiwel** [comment]
Going to hold off merging this until after #58 has been merged. Just want to ensure that Dynamo changes are working correctly first.

**Pull request merged**
